### PR TITLE
Release 0.11.0

### PR DESCRIPTION
**Pull request by Kevsy**
#### What type of PR is this?

Add one of the following kinds:
* correction

#### What this PR does / why we need it:

Fixed typo in User Story, re-attempt to commit YAML

#### Which issue(s) this PR fixes:

Fixes #

#### Changelog input

```
 release-note

```

#### Additional documentation 

This section can be blank.

```
docs

```

### CODE REVIEW

**Review by javierlozallu — DISMISSED** [review]

**Comment by javierlozallu** [comment]
I think is the same YAML as in #27 but don't know why there are Linter Errors

**Comment by Kevsy** [comment]
@javierlozallu I have no idea - when. I fork the repository including the Workflow Action, I get no linter error...

The errors are all related to the generic error codes from Commonalities. YAML snippets are unfortunately not easily portable between YAML files, unlike e.g. XML.

Let me try again, thanks for your patience!

**Review by javierlozallu — APPROVED** [review]

**Comment by javierlozallu** [comment]
@Kevsy, as Megalinter is passed I'm merging this PR to have the relase ready for meta-fall
